prometheus-targets.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "labels": {
+      "job": "api-ui"
+    },
+    "targets": ["api-ui.ethical-metrics.dappnode:9090"]
```

Comment:
why send the server name metric to DMS?

Reply:
to allow easy data filtering by server name label